plugin/evm/customtypes/header_ext_test.go

```diff
@@ -175,7 +175,7 @@ func allFieldsSet[T interface {
 			case []uint8, []*Header, Transactions, []*Transaction, Withdrawals, []*Withdrawal:
 				assert.NotEmpty(t, f)
 			default:
-				t.Errorf("Field %q has unsupported type %T", field.Name, f)
+				assert.Failf(t, "Field %q has unsupported type %T", field.Name, f)
```

Comment:
```suggestion
				assert.Failf(t, "invalid type", "Field %q has unsupported type %T", field.Name, f)
```
This for some reason expects another field, I think